<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataSetPrediccion</class>
 <widget class="QMainWindow" name="DataSetPrediccion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; vertical-align:sub;&quot;&gt;Predicción con modelo ajustado (Regresión múltiple)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Selecciones el mes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Selecciones el pais</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gasto de publicidad ($)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtpublicidad">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ingrese los siguientes valores para la prediccion de ingresos:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ingreso ($)</string>
    </property>
   </widget>
   <widget class="QLabel" name="e1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="meses">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Enero</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Febrero</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Marzo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Abril</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mayo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Junio</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Julio</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Agosto</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Septiembre</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Octubre</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Noviembre</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Diciembre</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="paises">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>España</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Holanda</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bélgica</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Alemania</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Francia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gran Bretaña</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Irlanda</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Italia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Suecia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Noruega</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dinamarca</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Estados Unidos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Suiza</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Austria</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rusia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Países del Este</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Resto de Europa</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Colombia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Resto de América</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Resto del Mundo</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="e2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="e3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>501</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CargarM">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>400</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar Modelo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar modelo entrenado.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPVentas">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>450</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Predecir Ventas</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
